jetty-ee9/jetty-ee9-tests/jetty-ee9-test-client-transports/src/test/java/org/eclipse/jetty/ee9/test/client/transport/HttpClientContinueTest.java

```diff
@@ -677,17 +675,14 @@ public void onComplete(Result result)
         assertTrue(latch.await(5, TimeUnit.SECONDS));
     }
 
-    @ParameterizedTest
-    @MethodSource("transports")
-    public void testExpect100ContinueWithTwoResponsesInOneRead(Transport transport) throws Exception
+    @Test
+    public void testExpect100ContinueWithTwoResponsesInOneRead() throws Exception
     {
-        assumeTrue(transport == Transport.HTTP || transport == Transport.HTTPS);
-
         // There is a chance that the server replies with the 100 Continue response
         // and immediately after with the "normal" response, say a 200 OK.
         // These may be read by the client in a single read, and must be handled correctly.
 
-        startClient(transport);
+        startClient(Transport.HTTP);
```

Comment:
No HTTPS testing anymore?

Reply:
No, because the code was not testing HTTPS, but HTTP clear text twice, so once is enough.